.github/workflows/test_ci.yml

```diff
@@ -67,14 +67,14 @@ jobs:
       - name: Run ENSIndexer runtime integrity checks
         env:
           # Note on managing below configuration with GitHub:
-          # We use private RPC URLs from GitHub Secrets to avoid rate limits.
-          # Public RPC URLs are used as fallbacks for repository forks
-          # that don't have the relevant secrets configured.
+          # We use private RPC URLs from GitHub Secrets.
+          # These must be configured in repository secrets for CI to work properly, especially for forks.
+          # Public (rate limited) RPC endpoints will not provide acceptable performance.
           PLUGINS: subgraph,basenames,lineanames,threedns
-          RPC_URL_1: ${{ secrets.MAINNET_RPC_URL || 'https://eth.drpc.org' }}
-          RPC_URL_10: ${{ secrets.OPTIMISM_RPC_URL || 'https://optimism.drpc.org' }}
-          RPC_URL_8453: ${{ secrets.BASE_RPC_URL || 'https://base.drpc.org' }}
-          RPC_URL_59144: ${{ secrets.LINEA_RPC_URL || 'https://linea.drpc.org' }}
+          RPC_URL_1: ${{ secrets.MAINNET_RPC_URL }}
+          RPC_URL_10: ${{ secrets.OPTIMISM_RPC_URL }}
+          RPC_URL_8453: ${{ secrets.BASE_RPC_URL }}
+          RPC_URL_59144: ${{ secrets.LINEA_RPC_URL }}
```

Comment:
@djstrong please take a look at the PR that introduced the fallback URLs pointing to public RPC services:
- https://github.com/namehash/ensnode/pull/884

Reply:
Thanks, I wasn't sure if in CI we should remove public RPC services too, so I will revert it.